ci-operator/step-registry/openshift-qe/installer/bm/day2/metallb/openshift-qe-installer-bm-day2-metallb-commands.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+
+if [ ${BAREMETAL} == "true" ]; then
+  bastion="$(cat /bm/address)"
+  # Copy over the kubeconfig
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion "cat ~/bm/kubeconfig" > /tmp/kubeconfig
+  # Setup socks proxy
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion -fNT -D 12345
+  export KUBECONFIG=/tmp/kubeconfig
+  export https_proxy=socks5://localhost:12345
+  export http_proxy=socks5://localhost:12345
+  oc --kubeconfig=/tmp/kubeconfig config set-cluster bm --proxy-url=socks5://localhost:12345
+fi
+
+oc config view
+oc projects
+
+# Install the MetalLB operator
+if [ ${METALLB} == "true" ]; then
+  cat << EOF| oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: metallb-system
+  labels:
+    openshift.io/cluster-monitoring: "true"
+EOF
+
+  cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: metallb-operator
+  namespace: metallb-system
+EOF
+
+  cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: metallb-operator-sub
+  namespace: metallb-system
+spec:
+  channel: stable
+  name: metallb-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+# Wait for the operator to be ready
+  until [ "$(kubectl get csv -n metallb-system | grep metallb-operator > /dev/null; echo $?)" == 0 ];
+    do echo "Waiting for MetalLB operator"
```

Comment:
This loop could go on forever, should we print out more debug?

Reply:
So, even if we print the output of:
```
kubectl get csv -n metallb-system
```

I guess at this moment it would not bring any relevant additional info. Do you have anything else in mind?

Reply:
at the moment no.. My concern is waiting forever, so in the vent we timeout due to waiting, it might be nice for just some debug in the step that failed.